src/components/common/Childnav.js

```diff
@@ -0,0 +1,87 @@
+
+import React from 'react';
+
+import { Link, useHistory } from 'react-router-dom';
+import { Button, Dropdown } from 'antd';
+import { MenuOutlined } from '@ant-design/icons';
+import { connect } from 'react-redux';
+import { global } from '../../state/actions';
+import { useOktaAuth } from '@okta/okta-react';
+
+const ChildMenu = props => {
+  const { push } = useHistory();
+  // const { authService } = useOktaAuth(); // https://github.com/okta/okta-react/blob/okta-react-4.0.0/README.md#migrating-from-3x-to-4x
+  // augment "oktaAuth" to behave like "authService"
+  const { authState, oktaAuth } = useOktaAuth();
+  oktaAuth.getUser = oktaAuth.token.getUserInfo;
+  oktaAuth.logout = oktaAuth.signOut;
+  oktaAuth.isAuthenticated = authState.isAuthenticated;
+  const authService = oktaAuth;
+  // end augmentation
+
+  const switchUsers = () => {
+    props.clearUsers();
+    push('/');
+  };
+
+  return (
+    <nav>
+      <ul>
+        <li>
+          <Link to="/child/dashboard">Home</Link>
+        </li>
+        <li>Help</li>
+        <li>
+          <button onClick={switchUsers}>Change User</button>
+        </li>
+        <li>
+          <button onClick={() => authService.logout()}>Log Out</button>
+        </li>
+        {/* Temporary navigation for user testing */}
+        {process.env.REACT_APP_ENV === 'development' && (
+          <>
+            <li>
+              <Link to="/child/join">Squad</Link>
+            </li>
+            <li>
+              <Link to="/child/match-up">Matchup</Link>
+            </li>
+          </>
+        )}
+      </ul>
+    </nav>
+  );
+};
+
+const Header = ({ clearUsers }) => {
+  const history = useHistory();
+
+  // hide navigation (hamburger) menu on certain pages
+  const showNav = () => {
+    switch (history.location.pathname) {
+      case '':
+      case '/login':
+        return false;
+      default:
+        return true;
+    }
+  };
+
```

Comment:
this function was left alone as it only relates to the hamburger menu on child pages and works well eliminating the hamburger on login pages.